benchmark/src/main/java/com/datatorrent/benchmark/cassandra/CassandraOutputOperator.java

```diff
@@ -48,4 +48,9 @@ protected Statement setStatementParameters(PreparedStatement updateCommand,
     return boundStmnt.bind(id++,tuple);
   }
 
+  @Override
+  public void deactivate()
```

Comment:
Can you move this to the Abstract parent class? Otherwise every implementation would have to include this method.